### PR TITLE
added more commands to German voice interpretation and made interpret…

### PR DESCRIPTION
**Pull request by kaikreuzer**
…er more robust

Signed-off-by: Kai Kreuzer <kai@openhab.org>

### CODE REVIEW

File: bundles/core/org.eclipse.smarthome.core.voice/src/main/java/org/eclipse/smarthome/core/voice/text/AbstractRuleBasedInterpreter.java [hunk 1]
```diff
@@ -480,10 +485,17 @@ protected String executeSingle(ResourceBundle language, String[] labelFragments,
                     if (oldState.equals(newState)) {
                         String template = language.getString("state_already_singular");
                         String cmdName = "state_" + command.toString().toLowerCase();
-                        String stateText = language.getString(cmdName);
+                        String stateText = null;
+                        try {
+                            stateText = language.getString(cmdName);
+                        } catch (Exception e) {
+                            stateText = language.getString("state_current");
```
Comment (maggu2810) [inline bundles/core/org.eclipse.smarthome.core.voice/src/main/java/org/eclipse/smarthome/core/voice/text/AbstractRuleBasedInterpreter.java]: For error a new const has been created above, for state_current the magic variable is used here.

Comment (maggu2810) [inline bundles/core/org.eclipse.smarthome.core.voice/src/main/java/org/eclipse/smarthome/core/voice/text/AbstractRuleBasedInterpreter.java]: Only one minor comment...

Comment (kaikreuzer) [inline bundles/core/org.eclipse.smarthome.core.voice/src/main/java/org/eclipse/smarthome/core/voice/text/AbstractRuleBasedInterpreter.java]: I did it the same way as the code looked like for the English version...

Comment (kaikreuzer) [inline bundles/core/org.eclipse.smarthome.core.voice/src/main/java/org/eclipse/smarthome/core/voice/text/AbstractRuleBasedInterpreter.java]: There are (already were) quite some other strings for which there is no constant defined in here - I did it similar to the state_already_* ones. But yes, all of these strings could be defined as constants throughout this class.

Comment (kaikreuzer) [inline bundles/core/org.eclipse.smarthome.core.voice/src/main/java/org/eclipse/smarthome/core/voice/text/AbstractRuleBasedInterpreter.java]: updated!

**Comment by kaikreuzer** [comment]
You are also allowed to merge then ;-)

**Comment by maggu2810** [comment]
The merge request builder (Travis CI) has not been finished till now.

**Comment by kaikreuzer** [comment]
Ah, ok!

**Pull request merged**